web/packages/teleterm/src/ui/Vnet/useVnetLauncher.tsx

```diff
@@ -131,12 +132,20 @@ export const useVnetLauncher = (): {
       const msg = msgParts.join(' ') + '.';
 
       if (isServer) {
-        const serviceInfo = await currentServiceInfo();
-        if (!serviceInfo.sshConfigured) {
-          openSSHConfigurationModal({
-            vnetSSHConfigPath: serviceInfo.vnetSshConfigPath,
-            host: addrToCopy,
-            onSuccess: () => notificationsService.notifyInfo(msg),
+        try {
+          const serviceInfo = await currentServiceInfo();
+          if (!serviceInfo.sshConfigured) {
+            openSSHConfigurationModal({
+              vnetSSHConfigPath: serviceInfo.vnetSshConfigPath,
+              host: addrToCopy,
+              onSuccess: () => notificationsService.notifyInfo(msg),
+            });
+            return;
+          }
+        } catch (err) {
+          notificationsService.notifyError({
+            title: isTshdRpcError(err) ? err.message : String(err),
```

Comment:
You can use `ensureError(err).message` that Grzegorz recently added instead of `isTshRpcError` which isn't necessarily applicable here (if the thrown error is a regular error, we'd probably also want to see its `message` property).